src/Nethermind.Int256/UInt256.cs

```diff
@@ -2206,7 +2226,7 @@ public object ToType(Type conversionType, IFormatProvider? provider) =>
         public ulong ToUInt64(IFormatProvider? provider) => System.Convert.ToUInt64(ToDecimal(provider), provider);
 
         [DoesNotReturn]
-        private static void ThrowDivideByZeroException() => throw new DivideByZeroException("y == 0");
+        private static void ThrowDivideByZeroException(string message) => throw new DivideByZeroException(message);
```

Comment:
I'd scrap the message; will add unneeded string load code; also is obvious what is zero

Reply:
Agreed, just didn't want to change the existing approach. Done in 9740891